challenges/classes.js

```diff
@@ -1,7 +1,33 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+
+class CuboidMaker {
+    constructor(props) {
+      this.length = props.length;
+      this.width = props.width;
+      this.height = props.height;
+    }
+  }
+  CuboidMaker.prototype.volume = function () {
```

Comment:
As we talked during 1:1 You want to get rid of prototype and move volume() and surfaceArea() as methods inside CuboidMaker class.